README.md

```diff
@@ -1,67 +1,88 @@
 # Wet Synthesis
-A collection of wrappers around solvers to simulate the wet-synthesis of Ni-Mn-Co Hydroxides 
 
-*Contact*: [Mohsen Shiea](mailto:mohsen.shiea@polito.it), 
+<!---introduction-start-c0e9f6f2-->
+
+A collection of [SimPhoNy](https://github.com/simphony/simphony-osp)
+[wrappers](https://simphony.readthedocs.io/en/v3.9.0/overview.html#fetch-data-from-a-database-run-a-simulation-and-immediately-store-the-results)
+around a CFD-PBE solver to simulate the wet-synthesis of Ni-Mn-Co Hydroxides.
+
+- CFD-PBE Wrapper: This wrapper prepares a simulation case folder based on the
+  user inputs and executes a solver for Computational Fluid Dynamics-Population
+  Balance Equation (CFD-PBE) simulation of NMC hydroxide precipitation. 
+
+- Compartment Wrapper: This wrapper is aimed at simulating the NMC hydroxide 
+  precipitation in a pilot CSTR reactor, by using the compartment model. 
+  Currently, it can be used only to divide the reactor into the compartments.
+
+*Contact*: [Andrea Querio](mailto:andrea.querio@polito.it), 
 [Daniele Marchisio](mailto:daniele.marchisio@polito.it), 
 [Antonio Buffo](mailto:antonio.buffo@polito.it) and 
 [Gianluca Boccardo](mailto:gianluca.boccardo@polito.it) from the 
-Multiphase Systems and Chemical Engineering research group, Politecnico di Torino.
+[Multiphase Systems and Chemical Engineering research group](https://www.disat.polito.it/research/research_groups/musychen), 
+[Politecnico di Torino](https://www.polito.it/).
 
-**Index**
-- [Wet Synthesis](#Wet-Synthesis)
-  - [CFD-PBE Wrapper](#CFD-PBE-Wrapper)
-  - [Compartment Wrapper](#Compartment-Wrapper)
-  - [Compatibility](#Compatibility)
-  - [Requirements](#Requirements)
-  - [Installation](#Installation)
-  - [Docker Container](#Docker-Container)
-  - [Usage](#Usage)
+<!---introduction-end-c0e9f6f2-->
 
-## CFD-PBE Wrapper
-This wrapper prepares a simulation case folder based on the user inputs and executes a solver for Computational Fluid Dynamics-Population Balance Equation (CFD-PBE) simulation of NMC hydroxide precipitation. 
-## Compartment Wrapper
-This wrapper is aimed at simulating the NMC hydroxide precipitation in a pilot CSTR reactor, by using the compartment model. Currently, it can be used only to divide the reactor into the compartments.
+## Installation
 
-## Requirements
-- [OSP core](https://github.com/simphony/osp-core)
-- [OpenFOAM](https://openfoam.org/)
-- [Ansys Fluent](https://www.ansys.com/products/fluids/ansys-fluent)
-- [7-zip](https://www.7-zip.org/)
+<!---installation-start-880c326a-->
 
-## Compatibility
+The wet synthesis wrappers require a working installation of 
+[OpenFOAM 8](https://openfoam.org/version/8/), 
+[SUNDIALS](https://github.com/LLNL/sundials) 
+([v6.1.1](https://github.com/LLNL/sundials/releases/tag/v6.1.1) is known to 
+work) and [7-zip](https://www.7-zip.org/). Follow the links to find 
+installation instructions for each tool.

```

Comment:
@AndreaQuerio This documentation assumes that only OpenFOAM is compatible, although theoretically, it looks like things shouldm work with Ansys Fluent too, but I cannot test this, so I left it out. If things work with Fluent, please modify the documentation yourself. Be careful, this is not the only place were you would have to mentiont Fluent.